starters/deno-oak-denodb/.env.example

```diff
@@ -8,4 +8,4 @@ DATABASE_PASSWORD=dbpassword
 DATABASE_PORT=5432
 
 
-CORS_ALLOWED_ORIGINS='http://localhost:3333'
```

Comment:
It's not necessary to use quotes in these files - they will be parsed correctly.